iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/auth/IJwtValidation.java

```diff
@@ -7,4 +7,5 @@
 @ImplementedBy(OpenIdConnectJwtValidation.class)
 public interface IJwtValidation {
     Boolean validateToken(String token) throws InvalidConfigurationException, ExternalDependencyException;
+    UserClaims getUserClaims(String token) throws NoAuthorizationException;
```

Comment:
Per feedback in https://github.com/Azure/pcs-config-dotnet/pull/95, change name to `NotAuthorizedException`

Reply:
make sense to me.